plopfile.js

```diff
@@ -38,10 +38,17 @@ module.exports = (
         return [
           {
             type: "addMany",
-            destination: `packages/components/${src}/{{pascalCase parentComponentName}}/subcomponents`,
+            destination: `packages/components/${src}/{{pascalCase parentComponentName}}/subcomponents/{{pascalCase componentName}}`,
             base: "plop-templates/basic-component/src",
             templateFiles: "plop-templates/basic-component/src/**/*.hbs",
           },
+          {
+            type: "addMany",
+            destination: `packages/components/${src}/{{pascalCase parentComponentName}}/subcomponents/{{pascalCase componentName}}/_docs`,
+            base: "plop-templates/basic-component/docs",
+            templateFiles:
```

Comment:
Nice add! However, this means that the `title` in the meta will be incorrect, so maybe we should also add a slight change to the template?

Example here https://github.com/cultureamp/kaizen-design-system/pull/4009/files#diff-a14030f4b4431c8cc6e6b098367fcc4173ef6413c59aecbd6594196ed1a367f1L60-L62

Reply:
Nice catch, I'll make that amendment and we should be hunky dory 🕺

Reply:
I've made the change while you're away, so we can use it for migrations :)